dask/array/slicing.py

```diff
@@ -1240,3 +1240,38 @@ def cached_cumsum(seq, initial_zero=False):
     if not initial_zero:
         result = result[1:]
     return result
+
+
+def take_along_axis(arr,indices,axis=None):
```

Comment:
We'd like to match NumPy's signature for this function, which doesn't have any default values

```suggestion
def take_along_axis(arr,indices,axis):
```

Reply:
After removing the none portion I think that `if axis is None:` is also not required but this code line is present in numpy, shall I remove both the things?

Reply:
I think instead of putting `if axis is None` we can put `if axis ==0` and remove the `axis=None` from line 1245. Is it correct?